src/Square.kt

```diff
@@ -0,0 +1 @@
+class Square(width: Int) : Rectangle(width, width)
```

Comment:
Inheritance detected. To fulfill Liskov Substitution Principle the class `Rectangle` should be replaceable by `Square` without causing unexpected behavior. Are you sure that Liskov Substitution Principle is fulfilled?